sdl_android/src/main/java/com/smartdevicelink/api/PermissionManager/PermissionElement.java

```diff
@@ -0,0 +1,58 @@
+package com.smartdevicelink.api.PermissionManager;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.protocol.enums.FunctionID;
+
+import java.util.List;
+
+/**
+ * PermissionElement holds the RPC name that the developer wants to add a listener for.
+ * It also holds any permission parameters for that RPC that the developer wants to track as well.
+ */
+public class PermissionElement {
+    private FunctionID rpcName;
+    private List<String> parameters;
+
+    /**
+     * Create a new instance of PermissionElement
+     * @param rpcName
+     * @param parameters
+     */
+    public PermissionElement(@NonNull FunctionID rpcName, List<String> parameters){
+        this.rpcName = rpcName;
+        this.parameters = parameters;
+    }
+
+    /**
+     * Get the RPC name
+     * @return FunctionID value represents the RPC name
+     */
+    public FunctionID getRPCName() {
+        return rpcName;
+    }
+
+    /**
+     * Set the RPC name
+     * @param rpcName
+     */
+    public void setRPCName(@NonNull FunctionID rpcName) {
```

Comment:
Does it ever make sense to set the rpc name or the parameters after the creation of the object? If not, these setters should be removed and the variables made final.